src/commands/list.cr

```diff
@@ -14,19 +14,21 @@ module Shards
 
       private def list(dependencies, level = 1)
         dependencies.each do |dependency|
-          resolver = dependency.resolver
-
-          # FIXME: duplicated from Check#verify
-          unless _spec = resolver.installed_spec
+          installed = Shards.info.installed[dependency.name]
```

Comment:
```suggestion
          installed = Shards.info.installed[dependency.name]?
```

Reply:
Thanks @Sija good catch! I added a spec to check for that condition